ocs_ci/utility/utils.py

```diff
@@ -2063,10 +2063,26 @@ def clone_repo(url, location, branch="master", to_checkout=None):
         location (str): path where the repository will be cloned to
         branch (str): branch name to checkout
         to_checkout (str): commit id or tag to checkout
+        clone_type (str): type of clone (shallow, blobless, treeless). If any
+            parameter is passed other than above 3 mentioned, its considered as normal clone.
```

Comment:
I would rather be more specific and tell user to pass None if they want normal clone, or just specify normal as one of the option.